### ISSUE TITLE
Create dataModel for handling Trips.

### ISSUE THREAD

**Issue by Pabluchenkoo** [ISIS3510-202410-Team11/Swift#26]

**Comment by Pabluchenkoo** [comment]
<img width="1102" alt="Screenshot 2024-04-21 at 9 14 46 PM" src="https://github.com/ISIS3510-202410-Team11/Swift/assets/66105423/29eaf9c2-7acf-49bc-8eb3-45ae7e356d05">

**Comment by Pabluchenkoo** [comment]
Let me know if you have comments @CristianCaro915. Great meeting! super productive!

**Comment by CristianCaro915** [comment]
Agree, it was great. We will see how difficult that route list is. For the next  iterations we should model the pricing process
